### DISCUSSION TITLE
Can we disable double-click to hide technical names on lists? It makes it hard to use double-click to select/highlight the whole field name.

### DISCUSSION THREAD

**Discussion by codaroma** [arnoudkooi/SN-Utils#505] [Ideas]
I like to use the show technical names slash command in lists to copy the backend field name to a script. The easiest way to select/highlight the whole field name (once it has appeared) (before copying with Ctrl-C) is to double click on it (it is easier than click and drag of mouse pointer) but double-clicking on the field technical name in a list causes it to disappear and be removed. Is it possible to disable the double-click hide?

**Comment by arnoudkooi** [comment]
I will exclude this from the doubleclick to toggle. As you can see in other issues/discussion, I'm very hesitant to add options...

**Reply by CyrusMcCleery** [reply]
While we're on this topic, can you also add the Find Next / Previous buttons in the script editor to the exclude list?

**Comment by Anduh** [comment]
The **Double-click Toggle**-option was included in the v.10.0.0.0 launch (April 2026).

Setting found under **Settings** > **Developer Tooling** > **Double-click Toggle**

Disabling it will stop the double click from both showing and hiding technical names, and disables it on lists & forms, so isn't exactly what you asked for.
- In [Discussion #624](https://github.com/arnoudkooi/SN-Utils/discussions/624) it had been pointed out that a general toggle had been requested multiple times, so with the big v10 release, the general toggle was added as an option.
- The v10 introduces a lot of new setting options for enabling/disabling certain features, and it's to the point that I can get why he didn't make all of them as granular to behave differently between list/form and also disable a toggle setting only in one direction.

**If you still want** `/tn` **quickly accessible to turn it on while keeping the double-click toggle disabled, there are multiple options:**
- Custom Open Shortcut: can bind one command to it, and then in browser extension keyboard shortcuts, you can define a keyboard shortcut to it
- (SN Utils Pro) you can make a dedicated [keyboard shortcut](https://snutils.com/docs/guide/slash-commands/shortcuts) for `/tn`, or add it to the [new Middle-click Action](https://snutils.com/docs/guide/lists-forms-ui/quick-ring-middle-click#middle-click-quick-action)
- (SN Utils Pro) or add it to the [Quick Ring](https://snutils.com/docs/guide/lists-forms-ui/quick-ring-middle-click#quick-ring)

Personally, I've added `/tn` to the Quick Ring, but also made 4 keyboard shortcuts I use even more frequently, to the point they are second nature to me after having used them for months.

(if the release of the **Double-click Toggle** is good enough for your usecase, you could close this discussion as resolved)
